examples/javascript/rgblcd.js

```diff
@@ -22,13 +22,13 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-var m = require('mraa'); //require mraa
+const mraa = require('mraa'); //require mraa
 
-x = new m.I2c(0)
-x.address(0x62)
-x.writeReg(0, 0)
-x.writeReg(1, 0)
+i2cDevice = new mraa.I2c(0)
```

Comment:
1. Let's add closing semicolons to all lines to align with the style used in this PR
1. This one should get `let` to follow the logic and style of this PR